src/in_process_server.cpp

```diff
@@ -1166,6 +1251,17 @@ class wlcs::Client::Impl
             me->data_device_manager = static_cast<struct wl_data_device_manager*>(
                 wl_registry_bind(registry, id, &wl_data_device_manager_interface, version));
         }
+        else if ("wl_output"s == interface)
+        {
+            auto wl_output = static_cast<struct wl_output*>(
+                wl_registry_bind(registry, id, &wl_output_interface, version));
+            auto output = std::make_unique<Output>(wl_output);
+            wl_output_add_listener(wl_output, &Output::listener, output.get());
+            me->outputs.push_back(move(output));
```

Comment:
We will, at some point, need to handle the `wl_output`s going away in `global_remove`, but that can be for later.